static/main.css

```
@@ -120,6 +123,9 @@ p {
 		max-width: initial;
 		max-height: calc( 100% - 150px );
 	}
+	.header h1 {
```

All h1s will be in the header...

```suggestion
	h1 {
```

Also could be good to shift this up to near the top of the file.